muted-tests.yml

```diff
@@ -359,6 +359,9 @@ tests:
 - class: org.elasticsearch.smoketest.MlWithSecurityIT
   method: test {yaml=ml/data_frame_analytics_cat_apis/Test cat data frame analytics single job with header}
   issue: https://github.com/elastic/elasticsearch/issues/125642
+- class: org.elasticsearch.repositories.blobstore.testkit.analyze.GCSRepositoryAnalysisRestIT
+  method: testRepositoryAnalysis
+  issue: https://github.com/elastic/elasticsearch/issues/125668
```

Comment:
I need to un-mute this anyway, any objection if we don't revert this bit?

Reply:
Seems fine to me.

Reply:
Sorry my comment maybe ambiguous. I meant it's ok to not revert since this is much simpler revert.

Reply:
Errr it got muted again 51ed156f09ff170ab1b25a4e542173ad6a48c4b7 😅

Reply:
Yeah I haven't got the client revert through the pipeline yet. I probably should have done them in the opposite order (client then logging).